### DISCUSSION TITLE
Playlist limit

### DISCUSSION THREAD

**Discussion by Sevinfolds** [navidrome/navidrome#3357] [Q&A]
Is there a limit to the number of tracks in a playlist? One of my playlists contains almost 1500 tracks, it does not start in the web interface (0.53.3), in any browser it says "something went wrong". In the Tempo application, this playlist works without problems. Maybe there are recommendations for the maximum number of tracks? What could be the reason for this?

**Comment by deluan** [comment]
There is no limit in playlist sizes, and they are paginated in the UI (pages of 100), so even extremely large playlists should have no issues showing up in the UI.

Please open the browser's inspect window and post any errors you may see in the console.
